moped-editor/src/views/projects/projectView/ProjectNameEditable.js

```diff
@@ -1,209 +1,40 @@
 import React, { useState } from "react";
-import makeStyles from "@mui/styles/makeStyles";
-import {
-  Box,
-  Grid,
-  Icon,
-  Snackbar,
-  TextField,
-  Typography,
-} from "@mui/material";
-import { useMutation } from "@apollo/client";
-import { Alert } from "@mui/material";
-import { UPDATE_PROJECT_NAME_QUERY } from "../../../queries/project";
+import { Snackbar, Alert } from "@mui/material";
 
-/**
- * GridTable Style
- */
-const useStyles = makeStyles((theme) => ({
-  root: {
-    width: "100%",
-  },
-  paper: {
-    width: "100%",
-    marginBottom: theme.spacing(2),
-  },
-  editIcon: {
-    cursor: "pointer",
-    marginLeft: "8px",
-  },
-  editIconConfirm: {
-    cursor: "pointer",
-    marginTop: "16px",
-    fontSize: "2rem",
-  },
-  titleDisplayField: {
-    display: "flex",
-    cursor: "pointer",
-  },
-  titleEditField: {
-    "font-size": "1.5rem",
-    "font-weight": "bold",
-  },
-  boxField: {
-    width: "100%",
-  },
-  titleId: {
-    alignSelf: "center",
-  },
-}));
+import ProjectNameForm from "./ProjectNameForm";
+import ProjectName from "./ProjectName";
 
 const ProjectNameEditable = (props) => {
-  const classes = useStyles();
-
   const DEFAULT_SNACKBAR_STATE = {
     open: false,
     message: null,
     severity: "success",
   };
 
-  const initialProjectName = props?.projectName;
-  const [projectName, setProjectName] = useState(initialProjectName);
-  const [projectNameBeforeEdit, setProjectNameBeforeEdit] =
-    useState(projectName);
-  const [isEditing, setIsEditing] = useState(props?.isEditing ?? false);
+  // used to control the snackbar's contents and visibility
```

Comment:
ohhhh "used to" as in "is used to" not "used to" like in the past - sorry my reading is not on point today! 😅

Reply:
I believe it is; it's passed down into the Snackbar component. If I'm missing something, let's open up an issue and I'll adjust it, but in the interest of the release tomorrow and this passing otherwise, I'm going to leave this here. Thank you!

Reply:
cool - thanks for taking a look 🙏